assets/translations/en-us.yaml

```diff
@@ -999,6 +999,8 @@ cluster:
     label: Cluster Name
     placeholder: A unique name for the cluster
   machineConfig:
+    banner:
+      updateInfo: Create a new pool to update machine configs
```

Comment:
I'm sure there's a more elegant way to relay this message... I'm open to suggestions 

![image](https://user-images.githubusercontent.com/835961/128767681-b2fae511-ed88-41c4-9b9f-0ca426343e2b.png)

Reply:
I think it's okay. @lvuch do you have any thoughts?